device.go

```diff
@@ -61,6 +61,7 @@ type DeviceResponse struct {
 	LogsChannel                interface{}   `json:"logs_channel,omitempty"`
 	IsLockedUntil              interface{}   `json:"is_locked_until__date,omitempty"`
 	IsAccessibleBySupportUntil interface{}   `json:"is_accessible_by_support_until__date,omitempty"`
+	MACAddress                 interface{}   `json:"mac_address,omitempty"`
```

Comment:
Have you checked what type they actually return in the API?

It would be good if we can set the actual type here instead of `interface{}`.

The reason for other fields being `interface{}` is mostly because I was not able to get them in a response from Balena in my tests to verify what it is

Reply:
I'll take a look this evening, I think it's a string